R/step_epi_shift.R

```diff
@@ -180,30 +148,49 @@ step_epi_shift_new <-
     )
   }
 
+
+
 #' @export
-prep.step_epi_shift <- function(x, training, info = NULL, ...) {
-  step_epi_shift_new(
+prep.step_epi_lag <- function(x, training, info = NULL, ...) {
+  step_epi_lag_new(
     terms = x$terms,
     role = x$role,
     trained = TRUE,
-    shift = x$shift,
+    lag = x$lag,
     prefix = x$prefix,
     default = x$default,
     keys = x$keys,
     columns = recipes_eval_select(x$terms, training, info),
     skip = x$skip,
-    id = x$id,
-    intended_direction = x$intended_direction
+    id = x$id
   )
 }
 
 #' @export
-bake.step_epi_shift <- function(object, new_data, ...) {
-  grid <- tidyr::expand_grid(col = object$columns, shift_val = object$shift) %>%
-    dplyr::mutate(newname = glue::glue(
-                          paste0("{object$prefix}","{abs(shift_val)}","_{col}")
-                        )
+prep.step_epi_ahead <- function(x, training, info = NULL, ...) {
+  step_epi_ahead_new(
+    terms = x$terms,
+    role = x$role,
+    trained = TRUE,
```

Comment:
Should be `trained = x$trained`.

Reply:
I actually think this is correct based on [recipes::prep.step_lag()](https://github.com/tidymodels/recipes/blob/93910b03a67a6243e52342d9b246ce30e72a0a8f/R/lag.R#L88).

Reply:
I stand corrected then.